### PR TITLE
Upgrade deps

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

**Comment by chenrui333** [comment]
cc @rakyll

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+dist: bionic
```
Comment (rakyll) [inline .travis.yml]: Why is this change?

Comment (chenrui333) [reply .travis.yml]: That is Ubuntu 18.04 LTS, which is latest and greatest :)

Comment (chenrui333) [reply .travis.yml]: Just thought it is nice to use the latest though.

**Comment by chenrui333** [comment]
what should I do to get this merged?

**Review by rakyll — APPROVED** [review]

**Pull request merged**